etc/docker/yarn-spark/Dockerfile

```diff
@@ -5,6 +5,9 @@ ENV ANACONDA_HOME=/opt/anaconda2
 ENV PATH=$ANACONDA_HOME/bin:$PATH
 ENV PYSPARK_PYTHON=$ANACONDA_HOME/bin/python
 
+RUN yum install -y gcc krb5-devel && \
+    yum clean all
```

Comment:
I had originally tried this approach and it still failed to build.  Only after using `;` between the two yum statements did they work.  If you found the above to work - that's definitely preferred.